app/src/main/res/values-pl-rPL/strings.xml

```diff
@@ -0,0 +1,546 @@
+    <string name="pref_title_default_quit_message">Wiadomość przy rozłączaniu</string>
+    <string name="pref_title_default_part_message">Wiadomość przy opuszczaniu kanału</string>
+
+
+    <!-- Interface settings -->
+
+    <string name="pref_header_interface">Interfejs</string>
+
+    <string name="pref_title_dark_theme">Ciemny motyw</string>
+    <string name="pref_summary_dark_theme">Zaaplikuj ciemny motyw interfejsu</string>
+    <string name="pref_title_primary_color">Główny kolor</string>
+    <string name="pref_title_accent_color">Kolor akcentu</string>
+    <string name="pref_title_font">Czcionka</string>
+    <string name="pref_title_font_size">Rozmiar czcionki</string>
+    <string name="pref_title_hide_join_part">Ukryj dołączenia/opuszczenia</string>
+    <string name="pref_summary_hide_join_part">Ukryj wszystkie wiadomości o dołączeniach i opuszczeniach</string>
+    <string name="pref_title_autocorrect">Włącz auto-korektę tekstu</string>
+    <string name="pref_summary_autocorrect">Zezwól na auto-korektę przez aplikację klawiatury</string>
+    <string name="pref_title_appbar_compact_mode">Ukryj nagłówek z nazwą sieci</string>
+    <string name="pref_title_message_format">Format wiadomości</string>
+    <string name="pref_title_nick_autocomplete">Uzupełnianie nicków/kanałów</string>
+
+    <string name="pref_title_nick_autocomplete_show_button">Pokaż przycisk</string>
+    <string name="pref_summary_nick_autocomplete_show_button">Pokaż przycisk uzupełniania nicku</string>
+    <string name="pref_title_nick_autocomplete_double_tap">Dotknij pola dwa razy</string>
+    <string name="pref_summary_nick_autocomplete_double_tap">Dotknij pola dwa razy aby uzupełnić nick lub kanał</string>
+    <string name="pref_title_nick_autocomplete_suggestions">Sugeruj nicki</string>
+    <string name="pref_summary_nick_autocomplete_suggestions">Pokaż sugestie nicków po wpisaniu dwóch znaków</string>
+    <string name="pref_title_nick_autocomplete_at_suggestions">Sugeruj nicki po \'@\'</string>
+    <string name="pref_summary_nick_autocomplete_at_suggestions">Pokaż sugestie nicków po znaku \'@\'</string>
+    <string name="pref_title_nick_autocomplete_at_suggestions_remove_at">Usuwaj \'@\'</string>
+    <string name="pref_summary_nick_autocomplete_at_suggestions_remove_at">Usuń znak \'@\' po wyborze sugestii</string>
+    <string name="pref_title_channel_autocomplete_suggestions">Sugeruj kanały</string>
+    <string name="pref_summary_channel_autocomplete_suggestions">Pokaż sugestie kanałów po wpisaniu znaku \'#\'</string>
+
+    <string-array name="pref_entries_font">
+        <item>Domyślna (Bezszeryfowa)</item>
+        <item>#</item>
+        <item>Monospace</item>
+    </string-array>
+    <string name="font_size_sample_text">To jest przykładowy tekst prezentujący rozmiar czcionki. Możesz go zmienić poniżej.</string>
+
+    <string-array name="pref_entries_appbar_compact_mode">
+        <item></item>
```

Comment:
I'd like a word of advice on this part because the translating app did it and it's confusing as hell to me